wtforms/fields/core.py

```diff
@@ -991,7 +995,8 @@ def __init__(
         super(FieldList, self).__init__(label, validators, default=default, **kwargs)
         if self.filters:
             raise TypeError(
-                "FieldList does not accept any filters. Instead, define them on the enclosed field."
+                "FieldList does not accept any filters."
+                " Instead, define them on the enclosed field."
```

Comment:
Be consistent with space before / after continuation. I prefer this after continuation style because it makes it more clear that there's a continuation, and feels more like the binary operator rule.

Reply:
Great catches !  I didn’t know that about the docstrings, but going forward I know Black’s default line length is 88. So do you want to set it at that or...?

Reply:
PEP 8 specifies different lengths for docstrings vs code. Use 72 for docstrings, 88 for code.

Reply:
Let's use the space after continuation for the reasons I gave above.